app/queries/appeals_with_no_tasks_or_all_tasks_on_hold_query.rb

```diff
@@ -54,6 +55,18 @@ def appeals_with_two_tasks_not_distribution
       .having("count(tasks) = 2 AND count(case when tasks.type = ? then 1 end) = 0", DistributionTask.name)
   end
 
+  # Confirm that all subtrees have an active task
+  def appeals_with_fully_on_hold_subtree
+    Appeal.where(id:
+      Task.left_outer_joins(:children).on_hold
+        .where.not(type: [RootTask.name, TrackVeteranTask.name, EvidenceSubmissionWindowTask.name, TimedHoldTask.name])
+        .group("tasks.id")
+        .having(
+          "count(case when children_tasks.status in (?) then 1 end) = 0",
+          Task.open_statuses
+        ).select(:appeal_id))
```

Comment:
In case there are multiple tasks in the same appeal with this problem.
```suggestion
        ).select(:appeal_id).distinct)
```